src/app/tabs/test-tab.js

```diff
@@ -141,37 +145,64 @@ module.exports = class TestTab extends ViewPlugin {
     // result.passingNum
     // result.failureNum
     // result.timePassed
-    this.testsSummary.hidden = false
+    if (!_err) {
+      this.testsOutput
+    }
     cb()
   }
 
+  cleanFileName (fileName) {
+    return fileName ? fileName.replace(/\//g, '_').replace(/\./g, '_') : fileName
+  }
```

Comment:
IMHO we should keep the pass here as well. the user can have same file name in two folders .